server/plugin/command.go

```diff
@@ -268,6 +270,16 @@ func (p *Plugin) handleCreateIncident(_ *plugin.Context, args *model.CommandArgs
 	return ""
 }
 
+func (p *Plugin) handleCreateRequest(_ *plugin.Context, args *model.CommandArgs, params []string, client Client, _ bool) string {
+	p.API.PublishWebSocketEvent(
+		constants.WSEventOpenCreateRequestModal,
+		nil,
+		&model.WebsocketBroadcast{UserId: args.UserId},
+	)
+
+	return ""
+}
```

Comment:
why return an empty string if that is of no use?

Reply:
please check this comment
https://github.com/mattermost/mattermost-plugin-servicenow/pull/78#discussion_r996857980